plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchMetadata.java

```diff
@@ -86,6 +86,7 @@
 import static com.google.common.collect.ImmutableSet.toImmutableSet;
 import static com.google.common.collect.Iterators.singletonIterator;
 import static io.airlift.slice.SliceUtf8.getCodePointAt;
+import static io.airlift.slice.SliceUtf8.lengthOfCodePoint;
```

Comment:
The commit title is too long. Please follow https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md

Also, I believe the issue isn't specific to Chinese, and other languages (e.g. Japanese) also has the same issue.

Reply:
Ok, I will change commit title later